prow/plugins/plugins.go

```diff
@@ -514,6 +515,25 @@ func (pa *PluginAgent) Config() *Configuration {
 	return pa.configuration
 }
 
+// ensureConfig ensure that the configuration
+// doesn't contain any nil value
+func ensureConfig(config *Configuration) error {
+	var errors []string
+	for repo, configuration := range config.Plugins {
+		for _, plugin := range configuration {
+			if plugin == "heart" && config.Heart == nil {
+				org := strings.Split(repo, "/")[0]
```

Comment:
It's valid to just have the org as a key here, so that wouldn't work for this print -- but you can probably just:

```go
errors = append(errors, fmt.Sprintf("unexpected nil config for plugin %s for %s", plugin, repo))
```

And it will format `org/repo` or `org` as necessary